packages/core/src/builder.class.ts

```diff
@@ -1130,7 +1130,15 @@ export class Builder {
 
     // Give the app a second to start up and set canTrack to false if needed
     if (Builder.isBrowser) {
-      this.setCookie(sessionStorageKey, sessionId, datePlusMinutes(30));
+      setTimeout(() => {
+        try {
+          if (this.canTrack) {
+            this.setCookie(sessionStorageKey, sessionId, datePlusMinutes(30));
+          }
+        } catch (err) {
+          console.debug('Cookie setting error', err);
+        }
+      });
```

Comment:
This is a similar approach to the one in this same file right below:

https://github.com/BuilderIO/builder/blob/1d88dd10836fa492704c2d6fce46d09c80ded6fe/packages/core/src/builder.class.ts#L1164-L1175